scripts/lint.sh

```diff
@@ -10,6 +10,7 @@ exitcode=0
 #  - readability/inheritance: cpplint complains about "override final" as being redundant, but it is not.
 #  - whitespace/braces: Does not work with {} initialization (see https://github.com/cpplint/cpplint/issues/204#issuecomment-1146769949).
 #  - build/include_subdir: Does not work well with our structure of using lib, test, and third_party directories.
+#  - whitespace/indent_namespace: Has a lot of false positives.
```

Comment:
One previous issue was resolved (https://github.com/cpplint/cpplint/issues/346). Is that something else?

If so, could you check what the issue and and re-open the ticket?
As I have not been tagged, I wasn't aware that this has been (apparently) resolved.

Reply:
I have added a comment and tagged the assignee. I do not have permission to reopen the ticket.

Reply:
We may need to open a new issue if the person is not responding.